terminate_program() {
clean_temp
show_err "Program terminated!"
exit 1
}
